README.md

```diff
@@ -98,4 +98,21 @@ Deploy the operator into the running cluster
 #### CI
 | *Command*           | *Description*                                                              |
 | ------------------- | -------------------------------------------------------------------------- |
-| `make setup/travis` | Downloads operator-sdk, makes it executable and copys to `/usr/local/bin/` |
\ No newline at end of file
+| `make setup/travis` | Downloads operator-sdk, makes it executable and copys to `/usr/local/bin/` |
+
+## Components versions
+
+Keycloak Operator supports the following version of key components:
+
+ | *Component*              | *Version/Tag*                                                          |
+ | ------------------------ | ---------------------------------------------------------------------- |
+ | `Keycloak`               | `jboss/keycloak:latest`                                                |
+ | `Red Hat Single-Sign-On` | `registry.redhat.io/redhat-sso-7-tech-preview/sso-cd-openshift:latest` |
+ | `Postgresql`             | `9.5`                                                                  |
+
+## Activating Red Hat Single-Sign-On
+
+It is possible to use Red Hat Single-Sign-On instead of Keycloak deployment. Pulling images
+from Red Hat Container Registry requires additional configuration steps
+(see [the manual](https://access.redhat.com/containers/?tab=images#/registry.access.redhat.com/redhat-sso-7-tech-preview/sso-cd-openshift)).
+In order to activate Red Hat Single-Sign-on, run the Operator with `--profile RHSSO` flag.
```

Comment:
Initially, I thought it would be easier from the productization point of view. The productized image will probably be based on different base layer and all we would need to is to pass additional flag in. 

After thinking about this a little bit more, I think you are right. We should introduce an additional flag (`profile`) to the CR Spec. This is much more flexible and it also enabled additional use case - a customer may use a productized Operator to run Keycloak (upstream).

Just to clarify - there is no danger of community users running a product here. Pulling RHSSO requires additional `Pull Secret` from Red Hat Container Registry, which is a manual installation step.